packages/solid-query/package.json

```diff
@@ -43,7 +43,7 @@
     "test:eslint": "eslint --ext .ts,.tsx ./src",
     "test:types": "tsc",
     "test:lib": "vitest run --coverage",
-    "test:lib:dev": "pnpm run test:lib --watch",
+    "test:lib:dev": "vitest watch --coverage",
```

Comment:
`vitest --watch` is a thing, `vitest run --watch` isn't - they have a dedicated `watch` command instead.